integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/detailquery/SearchModeTestCase.scala

```diff
@@ -111,22 +111,26 @@ class SearchModeTestCase extends QueryTest with BeforeAndAfterAll {
   }
 
   test("test lucene datamap with search mode") {
+    sql("set carbon.search.enabled = true")
     sql("DROP DATAMAP IF EXISTS dm ON TABLE main")
     sql("CREATE DATAMAP dm ON TABLE main USING 'lucene' DMProperties('INDEX_COLUMNS'='id') ")
+    sql(s"SELECT * FROM main WHERE id='100000'").show()
     checkAnswer(sql("SELECT * FROM main WHERE TEXT_MATCH('id:100000')"),
       sql(s"SELECT * FROM main WHERE id='100000'"))
     sql("DROP DATAMAP if exists dm ON TABLE main")
   }
 
   test("test lucene datamap with search mode 2") {
+    sql("set carbon.search.enabled = true")
```

Comment:
Is this required? Search mode is already started

Reply:
this one can be removed